ocaml/idl/datamodel.ml

```diff
@@ -3902,8 +3902,12 @@ let task =
 
 (** Many of the objects need to record IO bandwidth *)
 let iobandwidth =
-  [ field ~persist:false ~qualifier:DynamicRO ~ty:Float "read_kbs" "Read bandwidth (KiB/s)";
-    field ~persist:false ~qualifier:DynamicRO ~ty:Float "write_kbs" "Write bandwidth (KiB/s)" ]
+  let msg = "Disabled and replaced by RRDs -- see xcp-rrdd and /dev/shm/metrics" in
```

Comment:
This message is for the documentation of the XenAPI. Things like "xcp-rrdd" and "/dev/shm/metrics" are not relevant on that level, because the are details of the implementation of the API on the server-side.